### PR TITLE
Make payday gem compatible with Rails 6.1

### PR DESCRIPTION
**Pull request by st-misha**
## Overview
This PR bumps `i18n` version from `0.7.0` to `1.6.0`.
[Rails `activesupport` 6.1.7 gem](https://rubygems.org/gems/activesupport/versions/6.1.7) depends on `i18n >= 1.6, < 2`.
So to be able upgrade Rails to 6.1 this PR updates the `i18n` dependancy.

## Testing
The specs are ran automatically via [github action](https://github.com/sensortower/payday/blob/master/.github/workflows/ubuntu-latest.yml#L25-L27) that tests this build on different ruby versions.

### CODE REVIEW

File: lib/payday/pdf_renderer.rb [hunk 1]
```diff
@@ -302,7 +302,7 @@ def self.bold_cell(pdf, text, options = {})
     def self.number_to_currency(number, invoice)
       currency = Money::Currency.wrap(invoice_or_default(invoice, :currency))
       number *= currency.subunit_to_unit
-      number = number.round unless Money.infinite_precision
+      number = number.round unless Money.default_infinite_precision
```
Comment (stefanl-st) [inline lib/payday/pdf_renderer.rb:305]: Did you check that this is the same value?

File: .idea/modules.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (stefanl-st) [inline .idea/modules.xml]: Was this added by accident? Likely bad `.gitignore` for this folder. Always review your own PR please.

**Review by stefanl-st — APPROVED** [review]
Some comments and make sure you address them but approved.

File: lib/payday/pdf_renderer.rb (hunk 1 above)
Comment (st-misha) [reply lib/payday/pdf_renderer.rb:305]:
```
[DEPRECATION] `Money.infinite_precision` is deprecated - use `Money.default_infinite_precision` instead
```

Comment (stefanl-st) [reply lib/payday/pdf_renderer.rb:305]: Sure but did you actually run this and verify that it matches?

```
[PRODUCTION] HEAD irb(main):001:0> Money.infinite_precision
=> false
```

**Pull request merged**